script.py

```diff
@@ -0,0 +1,72 @@
+import subprocess
+import sys
+import boto3
+
+def mongodump(database,collections,outpaths):
+     for i in range(len(collections)):
+          command = ["mongodump", "--uri", database, "--collection", collections[i],"--out",outpaths[i]]
+          try:
+              subprocess.run(command, check=True)
+              print("Backup completed successfully.")
+          except subprocess.CalledProcessError as e:
+              print(f"Backup failed: {e}")
+
+def uploadtos3(outpaths,bucket,obj_paths):
+    try:
```

Comment:
unnecessary try except block, see the docs

Reply:
i did add try-except to boto3 client connection to see if connection is successful or not